addons/mass_mailing_slides/views/slide_channel_views.xml

```diff
@@ -21,7 +21,7 @@
             <xpath expr="//div[@name='action_channel_invite']" position="after">
                 <div role="menuitem">
                     <a name="action_mass_mailing_attendees" type="object"
-                       groups="mass_mailing.group_mass_mailing_user">
+                       groups="mass_mailing.group_mass_mailing_user" class="float-left pr-1">
```

Comment:
Your commit message is too long, do not write all the small changes you made, try to be synthetic 

e.g. this is useless, we see it when we read your PR
> Form View: new placeholder for "name"